### PR TITLE
New commands + Clean Up [Merge]

### PR DESCRIPTION
**Pull request by Irraquated**
Give me a few days to read through all the commands to see what can be "cleaned up"

Will rebase when done.

### CODE REVIEW

**Comment by Irraquated** [comment]
Also to be noted @Ecuacion the `move` command no longer works due to some main updates, I don't know the reason why but I tested this and it doesn't work.

**Comment by Ecuacion** [comment]
The `move` just force the bot to make a decision (to bypass lag), It's just a developing command really useless (like `checktour`). Anyway i'll review it.

Also thanks for the fixes and improvements :). Tag me when this is ready.

**Comment by Irraquated** [comment]
Can be merged @Ecuacion i've been busy lately and I don't think i'll be able to update as much as planned

File: languages/spanish.js [hunk 1]
```diff
@@ -242,6 +242,7 @@
 			'e21': 'El formato',
 			'e22': 'no es válido para retos',
 			'e31': 'No tengo equipos para retar en el formato',
+			'spf': 'Por favor especificar un formato', // I'm assuming this is wrong btw :P
```
Comment (Bryan-0) [inline languages/spanish.js]: ``Por favor especifique un formato`` <-- That would be better, I guess. (I'm spanish)

But, I think that ``Por favor especificar un formato`` could be good too.

**Comment by Ecuacion** [comment]
Closing this.
 - Command cleanup and reorganization done in e65d0727d46218a23364fc23829fcd72d49f25d4
 - Timed bans: using timeouts or intervals is dangerous, you must save the pointers and clear them on commands reloading. Also with a server restart, a timed ban is bypassed, because that implementation does not check user joining like autoban does. 
 - Global permissions like dynamic commands: this.can is for room permissions. Dynamic commands management, bot teams, start laddering, joining other rooms, etc are global things.

**Pull request closed without merging**